mappings/net/minecraft/SharedConstants.mapping

```diff
@@ -37,12 +37,15 @@ CLASS net/minecraft/unmapped/C_bcjitlto net/minecraft/SharedConstants
 		COMMENT Specifies whether Minecraft should use choice type registrations from the game's schema when entity types or block entity types are created.
 	FIELD f_ivxnowhr TICKS_PER_SECOND I
 	FIELD f_iwusaemq ENABLE_DEBUG_FEATURES Z
+	FIELD f_jawlepzf INCOMPATIBLE_WITH_ALL_OTHER_VERSIONS Z
 	FIELD f_jxryklep gameVersion Lnet/minecraft/unmapped/C_kszroxyx;
```

Comment:
This should be renamed to match the new class name

Reply:
What new class name? `GameAndWorldVersion`?

Reply:
yeah

Reply:
oh this was kinda my fault. its on the next line with `gameVersion`. your name was fine before